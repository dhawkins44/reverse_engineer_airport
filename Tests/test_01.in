4 6 10 5
0 3
0 0
1 0
2 0
3 0
0 1 10
0 2 10
0 3 10
1 2 10
1 3 10
2 3 10