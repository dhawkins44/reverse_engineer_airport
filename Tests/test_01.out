No suitable path from 0 to 3 exists
